euphonic/util.py

```diff
@@ -228,33 +253,42 @@ def custom_decode(dct):
         with open(filename) as fd:
             file_data = json.load(fd, object_hook=custom_decode)
     else:
-        raise ValueError(
-            f'No data files known for collection "{collection}". '
-            f'Available collections: '
-            + ', '.join(list(_reference_data_files)))
+        msg = dedent_and_fill(f"""
+            No data files known for collection "{collection}".
+            Available collections:
+            """ + ', '.join(list(_reference_data_files)),
+        )
+        raise ValueError(msg)
 
     if 'physical_property' not in file_data:
-        raise AttributeError('Data file does not contain required key '
-                             '"physical_property".')
+        msg = 'Data file does not contain required key "physical_property".'
+        raise AttributeError(msg)
 
     data = file_data['physical_property'].get(physical_property)
     if data is None:
-        raise ValueError(
-            f'No such collection "{collection}" with property '
-            f'"{physical_property}". Available properties for this collection'
-            ': ' + ', '.join(list(file_data['physical_property'].keys())))
+        msg =  dedent_and_fill(f"""\
+            No such collection "{collection}" with property
+            "{physical_property}". Available properties for this collection:
+            """ + ', '.join(list(file_data['physical_property'].keys())),
```

Comment:
```suggestion
            """ + ', '.join(file_data['physical_property']),
```
If this is a dict type